Lib/ufo2fdk/makeotfParts.py

```diff
@@ -122,11 +122,11 @@ def setupFile_menuName(self, path):
         ## get around this problem. sigh, old app bugs live long lives.
         if winCompatible != familyName or self.font.info.openTypeNameCompatibleFullName is not None:
             # windows
-            l = "l=%s" % winCompatible
+            l = "l=%s" % normalizeStringForPostscript(winCompatible)
```

Comment:
@typemytype: another question, why not use winStr() here, as you use macStr() below?